### ISSUE TITLE
Clicking on 'About' in the menu crashes the app.

### ISSUE THREAD

**Issue by Infi-Knight** [ashutoshgngwr/noice#11]
### Describe the bug
Clicking on 'About' in the menu crashes the app.

## To Reproduce
1. Open the Menu.
2. Click on 'About'

### Expected behavior
Should show some information about the app.

### Others
 - Device: MI A1, 
 - Android Version: [9]

**Comment by ashutoshgngwr** [comment]
### Corresponding stacktrace
```
kotlin.UninitializedPropertyAccessException: 
  at com.github.ashutoshgngwr.noice.fragment.SoundLibraryFragment.getMRecyclerView (SoundLibraryFragment.java)
  at com.github.ashutoshgngwr.noice.fragment.SoundLibraryFragment$mServiceConnection$1.onServiceConnected (SoundLibraryFragment.java)
  at android.app.LoadedApk$ServiceDispatcher.doConnected (LoadedApk.java:1725)
  at android.app.LoadedApk$ServiceDispatcher$RunConnection.run (LoadedApk.java:1757)
  at android.os.Handler.handleCallback (Handler.java:873)
  at android.os.Handler.dispatchMessage (Handler.java:99)
  at android.os.Looper.loop (Looper.java:193)
  at android.app.ActivityThread.main (ActivityThread.java:6762)
  at java.lang.reflect.Method.invoke (Method.java)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:493)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:858)
```

**Comment by ashutoshgngwr** [comment]
#12 works on Android Pie emulator but not on a physical device. Needs more work!

**Comment by ashutoshgngwr** [comment]
I went through the source code of [`AboutPage`](https://github.com/medyo/android-about-page/blob/master/library/src/main/java/mehdi/sakout/aboutpage/AboutPage.java). It does not seem to be causing the problem. Since I do not have a physical device with Android Pie and there is no other way to reproduce this behaviour, I did a workaround to avoid crash (#15). I'll leave this open until the exact origin of this bug is pinpointed and fixed.
